slides-battle-react/src/components/layout/Page.jsx

```diff
@@ -0,0 +1,113 @@
+import React from "react";
+import { CircularProgress } from "@material-ui/core";
+import styled, { css } from "styled-components";
+import { Typography } from "@material-ui/core";
+import img from "../../assets/images/vintage-mic.jpg";
+
+const Background = styled.div`

```

Comment:
This will generated css files ?

Reply:
When you use "styled" on a html component or custom one, it will create a random class name to create your css based on current information, in the app we can see classname like this "Background-xadiefer332eze" in order to ensure your className is not overrided by another one in one of your 94024920 files

So when someone will use the "Background" it will produce a div with a generated className which has this style linked to it